akd/src/errors.rs

```diff
@@ -183,6 +185,9 @@ impl fmt::Display for DirectoryError {
             Self::VerifyKeyHistoryProof(err_string) => {
                 write!(f, "{}", err_string)
             }
+            Self::InvalidEpoch(err_string) => {
```

Comment:
Yeah I think the test infra will swallow it up, which is fine for tests since we can debug them when needed. It's more for in-prod and if anything ends up user-facing, it'd be helpful in case the inner strings contains less helpful information